src/_pytest/main.py

```diff
@@ -367,6 +365,15 @@ def pytest_runtestloop(session: Session) -> bool:
     return True
 
 
+def _is_setuptools_project(path: Path) -> bool:
```

Comment:
This is a request for help: how can we easily access a pyproject.toml's contents to determine if setuptools is used?